Makefile

```diff
@@ -61,10 +65,6 @@ clean:
 	rm -rf build output dist examples/pacts
 
 deps: download_plugins
-	@echo "--- 🐿  Fetching build dependencies "
-	cd /tmp; \
-	go install github.com/mitchellh/gox@latest; \
```

Comment:
doesn't look to be used in this repo, and the repo is archived. May be a hangover from pact-go v1